server/@types/educationAndWorkPlanApi/index.d.ts

```diff
@@ -84,6 +84,22 @@ export interface paths {
     patch?: never
     trace?: never
   }
+  '/inductions/{prisonNumber}/induction-schedule': {
```

Comment:
There have been a few changes to the API recently - new endpoints etc.
For the most part the UI doesnt care about these because it doesnt call these new endpoints ... but the changes we are interested in are noted below .... 👇